templates/distribution/manifests/auth/kustomization.yaml.tpl

```diff
@@ -16,15 +16,23 @@ resources:
 {{- if ne .spec.distribution.modules.ingress.nginx.type "none" }}
   - resources/ingress-infra.yml
 {{- end }}
-
+{{- if and .spec.distribution.modules.auth.oidcKubernetesAuth.enabled .spec.distribution.modules.auth.oidcKubernetesAuth.trustedCA }}
+  - secrets/gangplank-trusted-ca.yml
+  - secrets/pomerium-trusted-ca.yml
+  - secrets/dex-trusted-ca.yml
```

Comment:
Do we actually need 2 secrets in the `kube-system` namespace (dex and gangplank)? Can't we use one?